commune_app/all_models/Chore.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from .users import User
+
+
+class Chore(models.Model):
```

Comment:
I can see that we have one more PR for add chore:
https://github.com/redhat-beyond/commune/pull/36
so maybe you can leave it out of this PR? wdyt?